requirements/prod.in

```diff
@@ -27,7 +27,7 @@ docutils==0.18.0
 envcat==0.1.1
 everett==3.1.0
 fluent.runtime==0.3.1
-fluent.syntax==0.17.0  # hard-pinned subdep to avoid compatibility issues
+fluent.syntax==0.18.1
```

Comment:
glad to see work happening on Fluent. Hopefully our tests are good enough to catch any incompatibilities. We do import and use some internal classes IIRC.

Reply:
Unit and integration tests all green, at least